src/main/java/com/synopsys/integration/detect/workflow/ArtifactoryConstantsHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.synopsys.integration.detect.workflow;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArtifactoryConstantsHelper {
+    private static final Logger logger = LoggerFactory.getLogger(ArtifactoryConstantsHelper.class);
+    
+    public static String getArtifactoryUrl() {
```

Comment:
I don't know how likely it is, but maybe it's worth it to memoize the value in case this method ends up being used in more places in the future.

Reply:
Are you saying have a variable that would store the result and only make the call if that variable wasn't initialized? Instead of making REST calls for every artifact that needs downloading? If so I think I like that. If not can you elaborate?

Reply:
I went with the above if you want to take a look at the latest commit.